system/doc/general_info/deprecations_23.inc

```diff
@@ -2,11 +2,7 @@
       <title>Crypto Old API</title>
       <p>The <seeguide marker="crypto:new_api#the-old-api">Old API</seeguide> is now deprecated
       and has also been
-<<<<<<< HEAD
```

Comment:
Shouldn't this merge conflict be resolved in an earlier commit?

Reply:
done